runtime/lua/vscode-neovim/autocmds.lua

```diff
@@ -0,0 +1,7 @@
+vim.api.nvim_create_autocmd("InsertLeave", {
```

Comment:
I think we should make every file a lua module. Require should not produce any action or side effects, it should only import the module, and setup should be called. This way in the future, we can support exposing data through the module that user code can import (without registering another auto command).

Reply:
Ok